src/CarbonAware.WebApi/src/Models/EmissionsForecastDTO.cs

```diff
@@ -4,23 +4,11 @@
 using System.Text.Json.Serialization;
 
 [Serializable]
-public record EmissionsForecastDTO
+public record EmissionsForecastDTO : EmissionsForecastBaseDTO
 {
     [JsonPropertyName("generatedAt")]
     public DateTimeOffset GeneratedAt { get; set; }
```

Comment:
Any chance you can add summaries to these since you're already changing the files